### PR TITLE
grpc init

### PR DESCRIPTION
**Pull request by YaoZengzeng**
Signed-off-by: Yao Zengzeng yaozengzeng@zju.edu.cn

### CODE REVIEW

**Comment by resouer** [comment]
`Boilerplate header is wrong`, you should check "Copyright 2016 The Kubernetes Authors"  part in your newly added files and make sure they are the same as others.

File: cmd/protoc-gen-gogo/protoc-gen-gogo.go [hunk 1]
```diff
@@ -0,0 +1,42 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
```
Comment (resouer) [inline cmd/protoc-gen-gogo/protoc-gen-gogo.go]: This is still wrong, right one: https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/kubelet.go#L1-L15

**Comment by resouer** [comment]
As the CI error said: you should run `hack/verify-flags-underscore.py -e > hack/verify-flags/exceptions.txt` to update the list.

**Comment by feiskyer** [comment]
@YaoZengzeng `pkg/kubelet/hyper/api/` should be renamed to `pkg/kubelet/hyper/types/`, `api.proto` should be renamed to `types.proto` and other scripts/reference of this package.  

The grpc pkg name should be kept same with hyperd's one, or else hyperd will complain the `PublicAPI` is not defined.

File: hack/install-protoc.sh [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  sudo yum install -y unzip
+fi
+
+# Install protoc
+cd /tmp
+curl -sSL https://github.com/google/protobuf/releases/download/v3.0.0/protoc-3.0.0-linux-x86_64.zip -o protoc-3.0.0-linux-x86_64.zip
+unzip protoc-3.0.0-linux-x86_64.zip
+sudo mv bin/protoc /usr/bin/protoc
+
+echo "protoc installed success."
```
Comment (resouer) [inline hack/install-protoc.sh:30]: Add new blank line here.

As well as other files with this nits.

Comment (YaoZengzeng) [reply hack/install-protoc.sh:30]: fix it.

**Comment by resouer** [comment]
lgtm

**Pull request merged**